content/workshops/nodejs-intro-to-socketio/_index.md

```diff
@@ -1,5 +1,28 @@
 ---
 _db_id: 31
-content_type: workshop
+content_type: topic
+prerequisites:
+  hard:
+  - topics/js-and-node-specific/node
+ready: true
 title: Nodejs intro to socketIO
----
\ No newline at end of file
+---
+
+What is Socket.io?
+
+Socket.io is Node.js library that enables real-time, bidirectional and event-based communication between web-clients and the server. It consists of a client-side library for the browser and a Node.js server. In simple english, refer [here](https://medium.com/re-write/using-node-js-expressjs-and-socket-io-for-real-time-web-applications-9ded8717108c).
+
+WebSockets:
+
+- [WebSockets](https://www.youtube.com/watch?v=vQjiN8Qgs3c)
```

Comment:
It's good to give people instructions when giving links. Otherwise some people just ignore them.

Say something like:

"Please make sure you understand everything covered in these resources"